redhat-developer/developers.redhat.com
# DEVELOPER-4905 File ref for svg icon

JIRA: https://issues.jboss.org/browse/DEVELOPER-4905

Changes: 

* Added file ref for svg icon to taxonomy
* Removed the media ref to taxonomy
* Updated the view to use the new field
* Cleaned up the view a little bit

Testing:

* Modify a Product Group term
* Verify the reference to the media field is gone
* Verify the reference to a standard file is present
* Verify the file will only accept svg files
* Save the changes
* Navigate to the new All products page
* Verify the icon displays in the "Expandable Product Group" assembly
